reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -198,7 +198,17 @@ You can pipe a JSON string to `ConvertFrom-Json`.
 
 ## NOTES
 
-The `ConvertFrom-Json` cmdlet is implemented using [Newtonsoft Json.NET](https://www.newtonsoft.com/json).
+This cmdlet is implemented using [Newtonsoft Json.NET](https://www.newtonsoft.com/json).
+
+Beginning in PowerShell 6, `ConvertTo-Json` attempts to convert strings formatted as timestamps to
+**DateTime** values. The converted value is a `[datetime]` instance with a `Kind` property set as
+follows:
+
+- `Unspecified`, if there is no time zone information in the input string.
+- `Utc`, if the time zone information is a trailing `Z`
+- `Local`, if the time zone information is given as a trailing UTC _offset_ like `+02:00`. The
+  offset is properly converted to the caller's local equivalent. The default output formatting does
```

Comment:
We should probably be careful about "local equivalent".  It depends on their system setting, right? This could infer it's based on where they are, not what their machine is configured to be.

Reply:
Fixed